### ISSUE TITLE
optional command to run when the template is rendered: do not execute multiple commands

### ISSUE THREAD

**Issue by rcarre** [hashicorp/consul-template#1336]
Hi everyone,
I would like to execute more that one command after templating. As detailed here under, I put a ';' in between the two commands to be executed.
Unfortunately, only the first one is run. When checking journalctl for nginxws service, it is never reloaded.
Thx for support.

### Consul Template version
consul-template v0.20.0 (b709612c)

### Configuration

```hcl

template {
  source = "/opt/application/nginxws/current/conf/nginxws.conf.tmpl"
  destination = "/opt/application/nginxws/current/conf/nginxws.conf"
  command = "chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf;service nginxws reload || true"
}

### Debug output
2020/01/24 15:46:26.825170 [DEBUG] (runner) diffing and updating dependencies
2020/01/24 15:46:26.825179 [DEBUG] (runner) health.service(webcomws|passing,warning) is still needed
2020/01/24 15:46:26.825187 [INFO] (runner) executing command "chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf; systemctl reload nginxws || true" from "/opt/application/nginxws/current/conf/nginxws.conf.tmpl" => "/opt/application/nginxws/current/conf/nginxws.conf"
2020/01/24 15:46:26.825217 [INFO] (child) spawning: chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf
2020/01/24 15:46:26.826524 [DEBUG] (runner) watching 1 dependencies
2020/01/24 15:46:26.826538 [DEBUG] (runner) all templates rendered
2020/01/24 15:46:26.827177 [DEBUG] (cli) receiving signal "child exited"
2020/01/24 15:46:51.751634 [DEBUG] (runner) receiving dependency health.service(webcomws|passing,warn

### Expected behavior

Both commnad to be executed in serial: 
1- chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf
2- service nginxws reload

### Actual behavior

Only first command is executed:
1- chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf

**Comment by eikenb** [comment]
Hey @rcarre, thanks for taking the time to submit this issue.

I don't think we'll need to add a feature to get you want you want. You can get this working currently with a couple options. First is via an external shell script. Put your commands together in it and run the script as the command. This should be pretty straightforward and you probably didn't want this due to the overhead of getting the shell script file there.

The other option, which will probably be the one you like, is to use an embedded sub-shell call to get what you have working. Taking your example command, changing it to this should work...
```
  command = "sh -c 'chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf; service nginxws reload || true'"
```
Basically combining your multiple shell commands into one by making them part of a single call to the shell. This will work for more complex things like if/then/else statements as well.

Let me know if you have any questions or don't think this will work.

Thanks.

**Comment by rcarre** [comment]
Many thanks @eikenb for your proposal.
Indeed, I already successfuly released the first workaround option i.e. external shell script.
However, I faced this "issue" when upgrading consul-template from version 0.10.0 to 0.20.0 meaning that consul-template 0.10.0 does spawn in serial the two command lines when consul-template 0.20.0 does only handle the first one.
This feature is definitively not a must have. However, Readme.md could be enhanced to make this item crystal clear ?
Thanks.

**Comment by eikenb** [comment]
@rcarre Thanks for the reply. I'm going to update this to a 'docs' issue and update the section on this with instructions on howto run multiple commands at once. I'll try to come up with a way to mention the change from 0.10.0 as well.
